src/python/scripts/run_gerp.py

```diff
@@ -28,45 +28,75 @@
     $ python run_gerp.py --msa_file alignment.mfa --tree_file tree.nw --gerp_exe_dir path/to/gerp/
 
 """
-
 import os
 import subprocess
+import argparse
+import json
 
-import argschema
 
+def add_ce_to_json(ce_file: str, json_file: str) -> None:
+    '''Enrich the json file with the constraint elements
 
-class InputSchema(argschema.ArgSchema):
-    """Calculates the Genomic Evolutionary Rate Profiling (GERP) of a multiple sequence alignment (MSA)."""
+    :param ce_file: file containing the constraint elements
+    :param json_file: json file with genome coordinate elevel
+    :return: None
+    '''
```

Comment:
In keeping with the rest of the python code base, I think it is preferable to write the docstrings in this format: https://github.com/Ensembl/ensembl-compara/blob/main/scripts/hal_alignment/hal_gene_liftover.py#L53-L62 with the `Args:` and `Returns:`